src/app/dashboard/admins/page.js

```
@@ -0,0 +1,217 @@
+    };
+
+    // Filter users based on the search query
+    const filteredUsers = users.filter(user =>
+        (user.name && user.name.toLowerCase().includes(searchQuery.toLowerCase())) ||
+        (user.lastName && user.lastName.toLowerCase().includes(searchQuery.toLowerCase())) ||
+        (user.email && user.email.toLowerCase().includes(searchQuery.toLowerCase()))
+    );
+
+    // Determine if a user is a special user (main admin)
+    const isSpecialUser = (user) => {
+        return (user.email && user.email.toLowerCase().includes('yassineova')) || 
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)